### DISCUSSION TITLE
Using Vercel features with Angular

### DISCUSSION THREAD

**Discussion by parthupadhye** [vercel/community#4873] [Idea]
Does anyone have experience on adding Vercel features/functions/integrations in an Angular application? 
Overall, my reading is that this is very possible. I got Node.js working in a minute. 

Any suggestions or ideas welcome.

**Comment by amyegan** [comment]
Hi @parthupadhye! Angular is supported on Vercel. There's a [boilerplate template](https://vercel.com/templates/angular/angular-boilerplate) and [example repo](https://github.com/vercel/vercel/tree/main/examples/angular) available to help you get started. 🙂

**Comment by parthupadhye** [comment]
Yes, I have that and much more working very smoothly. I have even set up
NodeJS within minutes.

I am wondering if it is just as easy to use Vercel KV, Postgres and other
integrations?

Thanks.

On Fri, Nov 17, 2023 at 4:15 PM Amy ***@***.***> wrote:

> Hi @parthupadhye <https://github.com/parthupadhye>! Angular is supported
> on Vercel. There's a boilerplate template
> <https://vercel.com/templates/angular/angular-boilerplate> and example
> repo <https://github.com/vercel/vercel/tree/main/examples/angular>
> available to help you get started. 🙂
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/orgs/vercel/discussions/4873#discussioncomment-7603263>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAB3QXZ5QVTLORTYWU3FJNLYE7HV5AVCNFSM6AAAAAA7OQVLUOVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM3TMMBTGI3DG>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 
Parth Upadhye [ 647 965 4542 ]

http://www.linkedin.com/in/parthupadhye
